AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.h

```diff
@@ -35,6 +35,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)flushQueue;
 
+/**
+ * Check any enqueued logs to send it to ingestion.
+ */
+- (void)checkPendingLogs;
```

Comment:
Move to the private header

Reply:
I am not sure why does the private header was imported in channelGroup. Fixed it. And added this one to the ChannelUnitProtocol.